src/main/resources/db/migration/V1_8__create_tables.sql

```diff
@@ -0,0 +1,163 @@
+create table region (
+	region_id varchar(16),
+	description varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	welsh_description varchar(256),
+	CONSTRAINT region_id_pk PRIMARY KEY (region_id)
+);
+
+create table building_location(
+	building_location_id varchar(16),
+	epimms_id varchar(16) NOT NULL,
```

Comment:
Hope all this tables should there in LRD api else it fails in Master.Else create scripts if those not there in aat.

Reply:
This has been take care already - https://github.com/hmcts/rd-location-ref-api/pull/266